### ISSUE TITLE
Introductions

### ISSUE THREAD

**Issue by chendaniely** [databridgevt/covid19#1]
Hi all:

As you are joining the team and getting acquainted with the project (and learning git).

Please do the following to add your name to the "Teams" section of the `README.md`.

1. `clone` this repository to your computer
2. Add your name and a short blurb about yourself in the `README.md` file
3. `add` the changes you made
4. `commit` your changes
5. `push` your changes back up to GitHub
    - Most likely this will block you from pushing. Just let me know about it this issue and I can add you as a contributor.

You should see your changes when you go to the repository page under the "Teams" section.
This task also serves as your understanding of Git and makes sure the settings on this repository are correct. So, please let me know if you run into issues.

**Comment by chendaniely** [comment]
yay @gitchrissi ! You're the first one! :tada:

**Comment by chendaniely** [comment]
Just realized. depending on when people `clone` the repository, when you try to `push` your changes you may be blocked for one of 2 reasons

1. Permissions (403 error): let me know your GH username so I can add you to this repository as a maintainer
2. Something about the remote having changes you don't have: if you keep reading the error message it's essentially telling you that you'd need to `pull` first before `push`ing again.
  - You may run into a merge conflict here depending on what lines were changed. Just let me know if you end up with problems here
  - The tl;dr is you need to open the `README.md` file and remove the `>>>>>>>`, `=====`, and `<<<<<` and clean up the entire file so you're happy with it. Then `add`, `commit`, and `push` again.

**Comment by chendaniely** [comment]
@mitchdolby your commits acd5408d606fa6e8c662998aef8a5452a626a39a and b26b25454429b163c75e9900d779cafc58a57898 didn't update the `README.md` file. It looks like you used the commit message for the text you were trying to put into the `README.md` file.

**Comment by chendaniely** [comment]
The `master` branch has been locked so, in order to update your name in the `README.md` file you also need to see the PR instructions in #4
